File: Lista-15/python/8.py
class Empregado:
    def __init__(self, nome, cargo, salario):
        self.nome = nome
        self.cargo = cargo
        self.salario = salario

class Empresa:
    def __init__(self, nome):
        self.nome = nome
        self.empregados = []

    def adicionar_empregado(self, empregado):
        self.empregados.append(empregado)

# Exemplo de uso
empregado1 = Empregado("João", "Desenvolvedor", 5000)
empresa = Empresa("Empresa X")
empresa.adicionar_empregado(empregado1)
print([(emp.nome, emp.cargo) for emp in empresa.empregados])

File: Lista-15/java/8.java
import java.util.ArrayList;
import java.util.List;

class Empregado {
    private String nome;
    private String cargo;
    private double salario;

    public Empregado(String nome, String cargo, double salario) {
        this.nome = nome;
        this.cargo = cargo;
        this.salario = salario;
    }

    public String getNome() {
        return nome;
    }

    public String getCargo() {
        return cargo;
    }
}

class Empresa {
    private String nome;
    private List<Empregado> empregados = new ArrayList<>();

    public Empresa(String nome) {
        this.nome = nome;
    }

    public void adicionarEmpregado(Empregado empregado) {
        empregados.add(empregado);
    }

    public List<Empregado> getEmpregados() {
        return empregados;
    }

    public static void main(String[] args) {
        Empregado empregado1 = new Empregado("João", "Desenvolvedor", 5000);
        Empresa empresa = new Empresa("Empresa X");
        empresa.adicionarEmpregado(empregado1);

        for (Empregado emp : empresa.getEmpregados()) {
            System.out.println(emp.getNome() + ", " + emp.getCargo());
        }
    }
}
